internals/src/Tizen.Inspection/Tizen.Inspection/Inspection.cs

```diff
@@ -0,0 +1,296 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeApi = Interop.Inspection;
+
+namespace Tizen.Inspection
+{
+    /// <summary>
+    /// The Inspection class is used for requesting client data and receiving inspection events.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public static class Inspection
```

Comment:
DO NOT use the same name for a namespace and a type in that namespace.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces

Reply:
Would you suggest to change namespace to Tizen.System?